pango/src/attr_list.rs

```diff
@@ -1,14 +1,14 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
-use crate::{AttrList, IsAttribute};
+use crate::{AttrList, Attribute};
 use glib::translate::*;
 use std::mem;
 
 impl AttrList {
     #[doc(alias = "pango_attr_list_change")]
-    pub fn change(&self, attr: impl IsAttribute) {
+    pub fn change(&self, attr: impl AsRef<Attribute>) {
```

Comment:
```suggestion
    pub fn change(&self, attr: impl Into<Attribute>) {
```

Maybe? Every `FooAttribute` can be `Into<Attribute>` via `upcast()`, and `T: Into<T>` blanket implementation exists

Reply:
Seems to work fine :-)

Reply:
You actually have to implement `impl From<AttrColor> for Attribute { ... }` etc, otherwise this only works for the literal `Attribute`

Reply:
Oups, I thought it was already implemented :)